apps/src/templates/sectionAssessments/CorrectAnswerIcon.jsx

```diff
@@ -1,19 +1,54 @@
-import React, {Component} from 'react';
+import React, {Component, PropTypes} from 'react';
 import color from "@cdo/apps/util/color";
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 
 const styles = {
+  main: {
+    border: 'none',
+    height: '30px',
+    marginTop: '20px',
+    marginRight: '80px',
+    paddingTop: '20px',
+    paddingLeft: '20px',
+    paddingRight: '10px',
+    paddingBottom: '10px',
+  },
   icon: {
     color: color.level_perfect,
-  }
+    float: 'left',
+    width: '10%',
+    fontSize: '25px',
+  },
+  text: {
+    color: color.black,
+    fontWeight: 'bold',
+    fontFamily: 'Gotham 3r',
+    fontSize: '14px',
+    float: 'left',
+    paddingRight: '40px',
+    width: '10%',
```

Comment:
since your cell width will be variable, using something like `10%` for width might make the width too small and cut off part of your content. I'd stick to something like `20px` or whatever you need.

Reply:
Fixed.  Thanks